frontend/src/pages/ShoppingListPage.tsx

```diff
@@ -2,23 +2,60 @@ import ItemForm from "../components/ItemForm.tsx";
 import {Item} from "../type/Item.tsx";
 import {useState} from "react";
 import GroupedItems from "../components/GroupedItems.tsx";
+import {useNavigate} from "react-router-dom";
 
 export default function ShoppingListPage(){
 
     const [items, setItems] = useState<{item: Item, quantity: string}[]>([]);
 
-    const handleSubmitNewItem = (item: Item, quantity: string) => {
+    const [shoppingListName, setShoppingListName] = useState("");
+
+    const handleGoShoppingButtonClick = () => {
+        const id: string = ""  // Get this from the POST API call
+        navigate("/shopping/" + id)
+    };
 
+    const handleSaveAndGoHomeButtonClick = () => {
+        // Add POST API call here
+        navigate("/")
+    };
+
+    const navigate = useNavigate();
+
+    const handleSubmitNewItem = (item: Item, quantity: string) => {
         setItems([...items, {item, quantity}]);
         console.log(items);
```

Comment:
Just a tip for future, if we want the console log to log the "correct" state for debugging, you can add it to a `useEffect block` that usually gets added just before the return section.
```
useEffect(() => {
    console.log(items);
}, [items]);

```

Reply:
awesome 🙌🏻